libs/barista-components/experimental/datepicker/src/timeinput.ts

```diff
@@ -190,15 +231,20 @@ export class DtTimeInput {
   /**
    * @internal Check if the minute input is valid and set the new value accordingly.
    */
-  _handleMinuteInputValidation(value: string): void {
-    if (isValidMinute(value)) {
-      this._minute = parseInt(value, 10);
+  _handleMinuteInputValidation(
+    value: string | number | null,
+    min?: string | number | null,
+    max?: string | number | null,
+  ): void {
```

Comment:
I don't think this needs to handle strings any more. You are passing `this.minute` this.minMinute`, this.maxMinute` which can all be either number | null`.